fearless/Common/Services/PayoutRewardsService/PayoutRewardsServiceProtocol.swift

```diff
@@ -7,6 +7,10 @@ protocol PayoutRewardsServiceProtocol {
     func fetchPayoutRewards(completion: @escaping PayoutRewardsClosure)
 }
 
-enum PayoutError: Error {
+enum PayoutError: LocalizedError {
```

Comment:
We use ErrorContentConvertible protocol for this. Once error adopted it can be displayed automatically through ErrorPresentable protocol